src/test/java/org/apache/commons/csv/CSVParserTest.java

```diff
@@ -427,7 +427,7 @@ public void testExcelFormat2() throws Exception {
     /**
      * Tests an exported Excel worksheet with a header row and rows that have more columns than the headers
      *
-     * @throws Exception
+     * @throws Exception Any exception can be thrown
```

Comment:
Not helpful. Tests like this one can throw `Exception` as a convention instead of throwing a potentially long list of exceptions which are not useful and just clutter up the method signatures. So either we should not have a Javadoc tag or if we have one, it should carry proper and useful documentation. In this case, I think that there is nothing interesting to say.

Reply:
OK, got it. Reverted